src/nupic/regions/SPRegion.py

```diff
@@ -159,29 +159,25 @@ def getConstraints(arg):
       description='Total number of columns (coincidences).',
       accessMode='Read',
       dataType='UInt32',
-      count=1,
-      constraints=''),
+      count=1 ),
 
     inputWidth=dict(
       description='Size of inputs to the SP.',
       accessMode='Read',
       dataType='UInt32',
-      count=1,
-      constraints=''),
+      count=1 ),
 
     spInputNonZeros=dict(
       description='The indices of the non-zero inputs to the spatial pooler',
       accessMode='Read',
       dataType='UInt32',
-      count=0,
-      constraints=''),
```

Comment:
Hi @vitaly-krugl
I really appreciate the 2nd pair of eyes on the code. I'll move forward as you recommended.

Remove empty "constraints" parameters from specs #2369 
https://github.com/numenta/nupic/issues/2369

If I misinterpreted how to fix the issue, I'd welcome feedback on how to approach the bug. 

Thanks!
m